frontend/schema.graphql

```diff
@@ -482,6 +483,11 @@ enum DbErrorCode {
   DUPLICATE
 }
 
+enum LexboxAudience {
+  LEXBOX_API
+  FORGOT_PASSWORD
```

Comment:
Looks like 2 are missing (Unknown and Send/Receive). Do you need to generate the schema again?

Reply:
Ahh yes good catch.